research/patient_lists.py

```diff
@@ -0,0 +1,31 @@
+from elcid import models
```

Comment:
Can we make this ./ridrti/* - it's not really a generic research service at all !